docs/bokeh/source/docs/reference.rst

```diff
@@ -49,3 +49,16 @@ be especially useful.
 :ref:`bokeh.settings`
     All Bokeh-related environment variables, which can be used to control
     things like resources, minification, and log levels, are documented here.
+
+
+Linking to the Bokeh documentation
+----------------------------------
+Developers of other packages that make use of Bokeh might want to use
+`intersphinx <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_
+to link to specific classes or functions in the Bokeh docs from their Sphinx
+documentation. The file needed for that is located at
+https://docs.bokeh.org/en/<version>/objects.inv
```

Comment:
This will render better with code fences around it I think
```suggestion
``https://docs.bokeh.org/en/<version>/objects.inv``
```

Reply:
I think either works. Without fences, it should create a direct link, I guess; but then nobody would want to look at the link by directly, so maybe that's not useful in this case.

Reply:
Right, but `https://docs.bokeh.org/en/<version>/objects.inv` (with the angle brackets) is not actually a valid link, I propose the code fence to suppress link creation mostly

Reply:
I already included that